.changeset/forty-trees-burn.md

```diff
@@ -0,0 +1,5 @@
+---
+'@microsoft/atlas-css': patch
```

Comment:
Just wanted to clarify my own understanding. Wouldn't this be a minor update instead of a patch, since it's just an addition rather than a fix? I don't think this needs to be changed though as it'll probably get lumped into another minor when it's published.

Reply:
changed

Reply:
minor👍